tsconfig.json

```diff
@@ -8,7 +8,9 @@
     "tools/challenge-auditor/index.ts",
     "tools/challenge-editor/**/*",
     "tools/challenge-helper-scripts/**/*.ts",
-    "tools/scripts/**/*.ts"
+    "tools/scripts/**/*.ts",
+    "client/src/__tests__/settings-button.test.js",
+    "client/src/__tests__/timeline-buttons.test.js"
```

Comment:
I am almost sure there is a better way to address these config needs; @ojeytonwilliams what do you think?

Reply:
Just to let you know these are auto generated.

Reply:
Elaborate the steps?

Reply:
Basically I ran these command in cli it was auto generated 

`npm run test:source client/src/__tests__/settings-button.test.js`
`npm run test:source client/src/__tests__/timeline-buttons.test.js`

Reply:
Those commands should not add any changes to the tsconfig.

We usually keep the test spec alongside the code component we are testing. You have chosen to keep it in `__tests__`, which is fine.

Why you're choosing not to use TypeScript here?